### PR TITLE
Create loadtest environment

### PR DESCRIPTION
**Pull request by RMcVelia**
### Context

New loadtest environment for publish in bat-prod space
- same infrastructure as prod
- manual deploy via deploy workflow or make (not automated)

### Changes proposed in this pull request

- Add new env to Makefile
- Add to deploy workflow 
- terraform updates
- Add required rails configuration files

Also added outside of this PR
- required gh secrets
- AZ KV secrets
- AZ storage container

### Guidance to review

- confirm KV app/infra secrets are as expected for a loadtest
- confirm infrastructure requirements
- if dfe-signin required url's will need to be added to dfe-signin
- deploy-plan loadtest

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Review by smneal — APPROVED** [review]
LGTM - pending check of the secrets

File: config/settings/loadtest.yml [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+find_url: https://find-loadtest.london.cloudapps.digital
 
 # URL of this app for the callback after sigining in
-base_url: https://publish-teacher-training-loadtest.london.cloudapps.digital
+base_url: https://publish-loadtest.london.cloudapps.digital
+
+search_ui:
+  base_url: https://find-loadtest.london.cloudapps.digital
+
+dfe_signin:
```
Comment (saliceti) [inline config/settings/loadtest.yml:12]: We agreed to now use DFE sign-in so can we remove this?

**Comment by smneal** [comment]
Discussed with @SamJamCul and @saliceti and updated the relevant secrets in Key Vault.

**Review by smneal — COMMENTED** [review]
Remove DfE Sign-in settings

File: config/settings/loadtest.yml (hunk 1 above)
Comment (smneal) [reply config/settings/loadtest.yml]: Removed DfE Sign-in settings

**Review by SamJamCul — APPROVED** [review]

**Pull request merged**